backend/src/main/scala/com/softwaremill/adopttapir/starter/content/ContentService.scala

```diff
@@ -0,0 +1,51 @@
+package com.softwaremill.adopttapir.starter.content
```

Comment:
it seems the code is not formatted - make sure that you have formatting enabled (should be automatic with intellij)

Reply:
I am not sure if I've found the correct option - do you mean enabling something like "reformat code on save"?

Reply:
That's what I have:

<img width="973" alt="image" src="https://user-images.githubusercontent.com/60503/187911815-54d3b3df-f65d-42a9-a7b5-da0b70968a57.png">